python_modules/dagster/dagster/_core/host_representation/external_data.py

```diff
@@ -109,6 +109,7 @@ class ExternalRepositoryData(
             ("external_job_datas", Optional[Sequence["ExternalJobData"]]),
             ("external_job_refs", Optional[Sequence["ExternalJobRef"]]),
             ("external_resource_data", Optional[Sequence["ExternalResourceData"]]),
+            ("external_asset_checks", Optional[Sequence["ExternalAssetCheck"]]),
```

Comment:
I wonder if the asset -> checks mapping should be here. I think the risk is if we extend checks to point to multiple assets in the future, so going with a list for now